src/components/TaskItemsList.tsx/TaskItemsList.tsx

```diff
@@ -7,11 +7,11 @@ type TasksItemListType = {
 
 const TasksItemList = ({ taskList }: TasksItemListType) => {
   return (
-    <>
+    <div style={{marginTop:20}}>
```

Comment:
Why inline styling and not class ? besides that formating - spaces missing {{}}

Reply:
If this is to keep distance between TaskItlems - why not use flex column and gap on a parent ?

Reply:
oh, that was for urgent solution, I'll fixx it